### PR TITLE
Base64 Encoding and Decoding

### PR DESCRIPTION
**Pull request by Vijay-Giri**

### CODE REVIEW

File: templatestore/frontend/src/components/base64.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
```
Comment (wilspi) [inline templatestore/frontend/src/components/base64.js:2]: rename the file to `templatestore/frontend/src/utils.js`

File: templatestore/views.py [hunk 2]
```diff
@@ -7,6 +7,17 @@
...
+
+
+def base64decode(template):
+    decoded_bytes = base64.b64decode(template)
+    return str(decoded_bytes, "utf-8")
+
+
+def base64encode(rendered_output):
+    encoded_bytes = base64.b64encode(rendered_output.encode("utf-8"))
+    return str(encoded_bytes, "utf-8")
```
Comment (wilspi) [inline templatestore/views.py:20]: similarly create a `utils.py` for this

File: templatestore/frontend/src/components/base64.js (hunk 1 above)
Comment (Vijay-Giri) [reply templatestore/frontend/src/components/base64.js:2]: ok

File: templatestore/views.py (hunk 2 above)
Comment (Vijay-Giri) [reply templatestore/views.py:20]: ok

**Pull request merged**